src/main/java/teammates/logic/core/FeedbackResponseCommentsLogic.java

```diff
@@ -361,11 +364,55 @@ private void verifyIsCoursePresent(String courseId) throws EntityDoesNotExistExc
         }
     }
 
-    private void verifyIsInstructorOfCourse(String courseId, String email) throws EntityDoesNotExistException {
-        InstructorAttributes instructor = instructorsLogic.getInstructorForEmail(courseId, email);
-        if (instructor == null) {
-            throw new EntityDoesNotExistException("User " + email + " is not a registered instructor for course "
-                                                + courseId + ".");
+    /**
+     * Verifies if comment giver is registered user/team of course. Since feedback participant comments can only be
+     * added through {@link teammates.ui.controller.FeedbackSubmissionEditSaveAction}, it is verified that comment giver
```

Comment:
No need to explain reason but state the assumption clearly. Actually one sentence should be enough.
```
Verifies if comment giver is registered user/team of course only.
```